rskj-core/src/main/java/org/ethereum/db/TrieKeyMapper.java

```diff
@@ -78,4 +79,10 @@
     public static byte[] storagePrefix() {
         return Arrays.copyOf(STORAGE_PREFIX, STORAGE_PREFIX.length);
     }
+
+    byte[] mapRskAddressToKey(RskAddress addr) {
```

Comment:
Yes, i didnt want to change static/instance, as i dont know if it has consequences, but i'll do it.

And for the access level, i put package to be accesible from Tests, as you said... and it could be protected to be accessed also by subclass.

Reply:
Done. Not static, since is more complicated to test, and its out of scope.
And if we wanted to extend the behaviour of this class, wouldnt it be better if its an instance method? @ajlopez